cluster_preflight_check/main.py

```diff
@@ -375,6 +377,10 @@ def run(context):
     setup_basic_context(context)
     parse_argument(context)
 
+    if not utils.is_root():
+        context.logger.fatal("{} can only be executed as user root!".format(context.name))
```

Comment:
If you use the logger instance without the `context` variable, you can just use `log.fatal(...)`.